app-crypt/libmd/files/libmd-1.0.4-fix-version-script-linker-support-detection.patch

```diff
@@ -0,0 +1,35 @@
+From e408786075b9540f76783f5c3ce87f6d1ece13cf Mon Sep 17 00:00:00 2001
```

Comment:
Can you put a link to the Gentoo bug + upstream bug + commit link all at the top of the patch? Thanks!

Reply:
Added the links below the patch description.